### ISSUE TITLE
[Bug Report] Rendering of outline over Label of Component

### ISSUE THREAD

**Issue by hidotp** [vuetifyjs/vuetify#8745]
### Environment
**Vuetify Version:** 2.0.10
**Vue Version:** 2.6.10
**Browsers:** Firefox 60.0
**OS:** Windows 10

### Steps to reproduce
See Link 

### Expected Behavior
Correct rendering of label, that the outline is fitted to the label

### Actual Behavior
The outline is rendered over the label, only after a forced rerender (such as a change of the value of the label) it is correctly rendered. This behavior can be observed in every component which uses the Label, Outlined and Placeholder. 

### Reproduction Link
<a href="https://codepen.io/coder63812/pen/NWKdzPJ" target="_blank">https://codepen.io/coder63812/pen/NWKdzPJ</a>

**Comment by hidotp** [comment]
Through some help, a workaround was found: https://codepen.io/coder63812/pen/qBWmNJJ,
the Textarea gets a key which is updated in the mounted() call.

**Comment by jacekkarczmarczyk** [comment]
see also https://github.com/vuetifyjs/vuetify/issues/8393#issuecomment-525010342

**Comment by egeersoz** [comment]
Any news on this issue? The provided workarounds work, but it's a considerable amount of effort to add them to every single location in our codebase where an `outlined` field is used inside a component (only to then have to remove the workaround when the issue is fixed).

**Comment by ascott18** [comment]
There's also a dead simple workaround to just put a background color on the label. You'll have to adjust the color to match whatever your background color is, of course. 

``` css
.v-text-field--outlined .v-label--active {
  background: white;
  padding-right: 4px;
}
```

**Comment by johnleider** [comment]
Related to https://github.com/vuetifyjs/vuetify/pull/9341

**Comment by johnleider** [comment]
fixed in #9341
